src/app/chromosome-service/chromosome.service.ts

```diff
@@ -1,19 +1,19 @@
 import { Injectable } from '@angular/core';
-import { Http } from '@angular/http';
+import { HttpClient } from '@angular/common/http';
+// tslint:disable-next-line:import-blacklist
 import { Observable } from 'rxjs';
 
 import { Chromosome } from './chromosome';
 
 @Injectable()
 export class ChromosomeService {
+  private readonly chromosomeUrl = 'chromosomes/';
 
-  chromosomeUrl = 'chromosomes/';
-
-  constructor(private http: Http) { }
+  constructor(private http: HttpClient) {}
 
   getChromosomes(): Observable<Chromosome[]> {
     return this.http
       .get(this.chromosomeUrl)
```

Comment:
This seems broken - shouldn't you append the chromosome URL to the base URL, as done elsewhere?

Reply:
I'm resolving these conversations as they have been moved to a different card in our Trello.